lib/agent/actions/fileretrieval/storage.js

```diff
@@ -9,14 +9,16 @@ var join      = require('path').join,
 var logger   = common.logger;
     watching = false;
 
-var NAMESPACE = "fr/";
+var storage_path = join(common.system.paths.config, 'commands.db');
+storage.init('files', storage_path, function(err) {
```

Comment:
I agree with you, but the problem is that I need to specify if i'm talking about commands or files when i'm calling some storage methods, because they can easily overlap when both storages are being used.